### ISSUE TITLE
Out of stock option description

### ISSUE THREAD

**Issue by SiR-DanieL** [woothemes/woocommerce#9030]
This option

![](http://cld.wthms.co/15WD4/5ZhOCrv1+)

seems to be misleading for our customers. Since in WC 2.4 if you have more than 20 variations it checks only after you select the options if it's in stock or not, they think that there's a bug when out of stock variations still show in the product page.

Can we add a description or reword it to explain that they will still show if there are more than 20 variations?

**Comment by mikejolley** [comment]
Should be clear enough; even if you have non-ajax variation, attributes from out of stock variations will still be hidden. Just confirmed. They will only show if used by out of stock and in stock variations at the same time.
